app/src/androidTest/java/com/github/sdpcoachme/profile/SelectSportsActivityTest.kt

```diff
@@ -64,10 +54,40 @@ open class SelectSportsActivityTest {
     }
 
     @Test
-    fun tickIconsInitiallyNotDisplayed() {
+    fun defaultParametersDisplayedCorrectly() {
         ActivityScenario.launch<SelectSportsActivity>(launchSelectSports).use {
+            composeTestRule.onNodeWithTag(TITLE).assertIsDisplayed().assertTextEquals(DEFAULT_TITLE)
+            composeTestRule.onNodeWithTag(DONE).assertIsDisplayed()
+            composeTestRule.onNodeWithTag(CANCEL).assertIsDisplayed()
             ROW_TEXT_LIST.forEach {
-                composeTestRule.onNodeWithTag(it.ICON).assertDoesNotExist()
+                composeTestRule.onNodeWithTag(it.ICON, useUnmergedTree = true).assertDoesNotExist()
+            }
+        }
+    }
+
+    @Test
+    fun parametersDisplayedCorrectly() {
```

Comment:
Maybe here as well, you could choose a more meaningful name such as "parametersDisplayedCorrectlyWhenSportsWereChosenAlready" so that we can already guess from the title what the test checks.